source/install/smtp-email-setup.rst

```diff
@@ -163,25 +163,8 @@ Checking your SMTP server is reachable
 -  You must run the following commands from the same machine or virtual
```

Comment:
It looks like the "following" commands have been removed from this section. Are there still commands that can/must be run? If not, perhaps something like this could work: 

- Ensure you are running commands from the same machine or virtual instance where mattermost/bin/mattermost is located. 
- Attempt to telnet to the email service to make sure the server is reachable.
- Ping the smtp server to see if a connection can be established.

Reply:
So the problem here, and why I made the change, is that we don't install telnet inside the docker containers so we can't advise using that to verify the connection. 

I think referring to ping is probably the best thing we can do at this point apart from telling people to install telnet themselves and test it that way.